dataland-frontend/src/components/resources/frameworkDataSearch/p2p/P2pModalColumnHeaders.ts

```diff
@@ -0,0 +1,7 @@
+export const p2pModalColumnHeaders = {
+  driveMixPerFleetSegment: {
+    vehiclesType: "Vehicles type",
+    driveMixPerFleetSegmentInPercent: "Percentage of drive mix per fleet segment",
```

Comment:
Change this header to "Share of alternative drive types per fleet segment" and also display the percentage symbol directly in the table:

![image](https://github.com/d-fine/Dataland/assets/35459207/e39d7219-8fb6-4780-96f2-750908fafa0c)

Reply:
changed

Reply:
ok